vignettes/articles/story-npe-background.Rmd

```diff
@@ -23,14 +23,14 @@ knitr::opts_chunk$set(
 )
 ```
 
-```{r, message=FALSE, warning=FALSE}
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
```

Comment:
Why `echo = FALSE` for this vignette but not the others?

Reply:
This is try to library some utility R package for display purpose, so we can non-echo it